tests/configs/basic_app/k8s_cronjobs.yml

```diff
@@ -128,3 +134,93 @@ spec:
             imagePullPolicy: 'Always'
             name: generic-container
           restartPolicy: OnFailure
+
+---
+
+kind: CronJob
+metadata:
+  name: generic-cronjob-with-success-history
+  labels: {
+    kubetools/name: generic-cronjob-with-success-history,
+    kubetools/project_name: generic-app,
+    kubetools/role: cronjob
+  }
+  annotations: {
+    app.kubernetes.io/managed-by: kubetools,
+    description: 'Run: [''generic-command'']'
+  }
+spec:
+  schedule: "*/1 * * * *"
+  startingDeadlineSeconds: 10
+  concurrencyPolicy: "Allow"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: generic-cronjob-with-success-history
+          labels: {
+            kubetools/name: generic-cronjob-with-success-history,
+            kubetools/project_name: generic-app,
+            kubetools/role: cronjob
+          }
+          annotations: {
+            app.kubernetes.io/managed-by: kubetools,
+            description: 'Run: [''generic-command'']'
+          }
+        spec:
+          containers:
+          - command: [generic-command]
+            containerContext: generic-context
+            env:
+            - {name: KUBE, value: 'true'}
+            image: generic-image
+            imagePullPolicy: 'Always'
+            name: generic-container
+          restartPolicy: OnFailure
+
+---
+
+kind: CronJob
+metadata:
+  name: generic-cronjob-with-fail-history
+  labels: {
+    kubetools/name: generic-cronjob-with-fail-history,
+    kubetools/project_name: generic-app,
+    kubetools/role: cronjob
+  }
+  annotations: {
+    app.kubernetes.io/managed-by: kubetools,
+    description: 'Run: [''generic-command'']'
+  }
+spec:
+  schedule: "*/1 * * * *"
+  startingDeadlineSeconds: 10
+  concurrencyPolicy: "Allow"
+  successfulJobsHistoryLimit: 3
```

Comment:
This should also not appear

Reply:
https://github.com/EDITD/kubetools/pull/166/commits/b60c09557b9a04a5ba8f7c486d8b8c36d424adb0